src/suspectplacesearcher/main.py

```diff
@@ -0,0 +1,131 @@
+# -*- coding: utf-8 -*-
+from geopy.distance import vincenty
+import requests
+import math
+
+
+class SuspectPlaceSearch(object):
+
+    ''' Return a dictonary containt information of a
+        suspect place arround a given position.
+        A suspect place is some company that match a suspect keyword
+        in a Google Places search.
+        input:
+            - lat : latitude
+            - lng : longitude
+        output: closest_suspect_place:
+            closest_suspect_place is a dict with the keys:
+                - name : The name of suspect place
+                - latitude : The latitude of suspect place
+                - longitude : The longitude of suspect place
+                - distance : The distance in meters between suspect place and the given lat, lng 
+                - address : The address  of suspect place
+                - phone : The phone of suspect place
+                - id : The Google Place ID of suspect place
+                - keyword : The Keyword that generate the place
+                            in Google Place Search
+    '''
+
+    def __init__(self, key):
+        self.GOOGLE_API_KEY = key
+
+    def search(self, lat, lng):
+
+        suspect_keywords = ["Acompanhantes", "Motel",
```

Comment:
@guilhermeprokisch's [said](https://github.com/datasciencebr/serenata-de-amor/pull/117#issue-187986747) he had _collected some of this keyword empirically_ — I guess he was listing them manually, trying them out and using the one that made sense according to theses manual tests.